src/BuiltInTools/dotnet-format/CodeFormatter.cs

```diff
@@ -279,5 +279,31 @@ private static async Task<Solution> RunCodeFormattersAsync(
             formattableDocuments.AddRange(sourceGeneratedDocuments);
             return (projectFileCount + sourceGeneratedDocuments.Count, formattableDocuments.ToImmutable());
         }
+
+        private static bool IsFantomasInstalled()
+        {
+            var processStartInfo = new ProcessStartInfo
+            {
+                FileName = "dotnet",
+                Arguments = "tool list -g",
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false,
+                CreateNoWindow = true
+            };
+
+            using var process = new Process { StartInfo = processStartInfo };
+            process.Start();
+            var output = process.StandardOutput.ReadToEnd();
+            process.WaitForExit();
+
+            return output.Contains("fantomas-tool");
```

Comment:
`fantomas-tool` is deprecated

The name of the tool is `fantomas` now.

![CleanShot 2024-11-10 at 18 01 26](https://github.com/user-attachments/assets/e5c0fa5f-f329-499e-b1dd-5692be647a3f)

Reply:
thanks!  github copilot workspaces gen this one, good catch